packages/py-api/py_api/controllers/url_shortener_controller.py

```diff
@@ -26,6 +27,7 @@ def delete_shortened_url(cls, endpoint: str) -> Dict[str, str] | JSONResponse:
             return JSONResponse(content={"message": "Endpoint wasn't found!"}, status_code=404)
 
     @classmethod
+    @handle_exception
```

Comment:
@Aleks1908 @IvanObreshkov 


This is the new decorator that we need to use to add the body to the exception message received by the end user. Honestly, I believe it will make our codebase more cluttered. I suggest that we remove the feature for displaying bodies. Since our APIs are not publicly accessible, in most debugging scenarios, you can already see the request body because you're the one making the request.

WDYT?

Reply:
Nice job creating the controllers. Also thank you for making the effort to write the decorator. In my opinion aside from our code base being more clutterd, people might forget to add it to the routes. Also for the Post/Put requests we are going to have a body validation via Pydantic, by creating Models. Now for the environments. Both locally and in prod these messages are useful (the url of the requst and the HTTP method), especially in prod. But wouldn't the Body validation of Pydantic keep us safe and prevent the need for displaying the body of the request in the exception message?

Reply:
I believe so. Essentially we can be even more strict with our Pydantic requirements in order to prevent the usage of incorrect values. For example, if we have a `verify` property on the body which can take only two values, we should enforce a check for those two types and throw a Pydantic exception if the passed value doesn't match none of the expected values. 

Having the body would've been helpful but I think we don't really need it right now. I prefer keeping our codebase consistent and easy to read for the backend newcomers.

Reply:
Yes, totally agree with you. If you want you can leave the controllers for the health routes. And remove the decorator in order to merge this PR, ot you could create a separate one. Also @Alex you could open an PR for #495

Reply:
I suggest that @Aleks1908 creates the controllers in his PR and I close this one out. It will help him understand the high level concept of controllers and routes.

Reply:
Deal🤝